### ISSUE TITLE
The kernel appears to have died. It will restart automatically

### ISSUE THREAD

**Issue by gaoexingyun** [keras-team/autokeras#1048]
I used StructuredDataClassifier to train the model but it prompted me that The kernel appears to have died. It will restart automatically. I have tried many times, but the same.
The following is my code:
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd
data = pd.read_csv("minmax.csv",header = None)
# divide the dataset into independant values and target value 
X = data[data.columns[0:41]] 
y = data[41]
import autokeras as ak
from sklearn.model_selection import cross_val_score
from sklearn.model_selection import StratifiedKFold
from sklearn.preprocessing import OneHotEncoder
kf = StratifiedKFold(n_splits=5,shuffle=True)
clf = ak.StructuredDataClassifier(max_trials=2)
for train_index, test_index in kf.split(X,y):
    X_train, X_test = X.iloc[train_index], X.iloc[test_index]
    y_train, y_test = y.iloc[train_index], y.iloc[test_index]
    clf.fit(X_train, y_train)
    # Predict with the best model.
    predicted_y = clf.predict(X_test)
    # Evaluate the best model with testing data.
